plugins/NitroSpoof/src/main/java/com/xinto/aliuplugins/nitrospoof/PluginSettings.kt

```diff
@@ -37,7 +39,19 @@ class PluginSettings(
             }
         }
 
+        val emptyToggle = Utils.createCheckedSetting(context, CheckedSetting.ViewType.SWITCH, "Enable Empty Character", "This allows you to embed your emote without the URL being present. Some servers do not allow empty character.").apply {
+		this.isChecked = settingsAPI.getBool("emptyChar", false)
+		this.setOnCheckedListener {
+			c: Boolean? -> if (PluginManager.isPluginEnabled("MoreHighlight")) settingsAPI.setBool("emptyChar", c!!) else { 
```

Comment:
I honestly don't understand this style of lambda declarations. Just don't.

Reply:
You mean the name of the argument or ?

Reply:
No, I mean placing the lambda arguments on a newline. The convention is the following:
```kt
this.setOnCheckedListener { c: Boolean -> 

}
```